app/static/src/scss/footer.scss

```diff
@@ -1,57 +1,68 @@
+/* SCSS for Footer */
 
-/* CSS for Footer */
+@mixin on-mobile{
+   @media screen and (width <= 640px){
+       @content;
+   }
+}
 
 
 .govuk-footer  {
-    padding-bottom: 40px;
- }
-
- .govuk-footer__tna-logo {
-    width: 100px;
- }
-
- .govuk-footer__meta {
-    margin: 0;
-    align-items: center;
-    gap: 40px;
- }
-
- .govuk-footer__meta-item {
-    display: block;
-    margin: 0;
- }
-
- .govuk-footer__inline-list-item {
-    margin-bottom: 10px;
- }
-
- @media screen and (width <= 640px) {
-    .govuk-footer__meta {
-        flex-direction: column;
-     }
-
-     .govuk-footer__link {
-        order: 2;
-     }
-
-     .govuk-footer__meta-item--grow {
-        flex-basis: auto;
-     }
-
-     .govuk-footer__inline-list-item  {
-        display:block;
-        margin-right: 0;
-        margin-bottom: 0;
-        text-align:center;
-        padding: 10px 0;
-     }
-
-     .govuk-footer__licence-logo{
-        margin:0 auto 10px;
-        display:block;
-     }
-
-     .govuk-footer__licence-description {
-        text-align: center;
-     }
- }
+   padding-bottom: 40px;
+
+
+&__tna-logo {
+   width: 100px;
+}
+
+&__meta {
+   margin: 0;
+   align-items: center;
+   gap: 40px;
+@include on-mobile {
+flex-direction: column;
+}
```

Comment:
formatting of curly braces looks off but we can enforce the formatting like we discussed by adding prettier to the precommit hooks in the repo:

Add the following to `.pre-commit-config.yaml`


```
- repo: https://github.com/pre-commit/mirrors-prettier
  rev: "v3.1.0"
  hooks:
    - id: prettier
```